membership_handlers.go

```diff
@@ -93,10 +94,19 @@ func (mh *membershipHandler) rolesChecker() (string, error) {
 	return "Error connecting to Bertha Authors", err
 }
 
-func (mh *membershipHandler) GoodToGo(writer http.ResponseWriter, req *http.Request) {
-	if _, err := mh.authorsChecker(); err != nil {
-		writer.WriteHeader(http.StatusServiceUnavailable)
+func (mh *membershipHandler) GTG() gtg.Status {
```

Comment:
Why doesn't the GTG check also call `authorsChecker`?

Reply:
I think it was assumed that one check would be enough as both are connecting to some Google Spreadsheet, but it would seem fair to me to be both, so I'm adding the other one also.